data/json/monsters/mammal.json

```diff
@@ -373,6 +376,7 @@
     "anger_triggers": [ "STALK", "FRIEND_ATTACKED", "PLAYER_WEAK", "HURT", "PLAYER_CLOSE" ],
     "placate_triggers": [ "MEAT" ],
     "death_function": [ "NORMAL" ],
+    "harvest": "animal_medium",
```

Comment:
Also will need to remove the FUR/BONES flags like you've done similarly on line 422